config/environment.js

```diff
@@ -52,8 +52,8 @@ module.exports = function(environment) {
     // ENV.APP.LOG_TRANSITIONS = true;
     // ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
     // ENV.APP.LOG_VIEW_LOOKUPS = true;
-    ENV.contractMetadata['networkId'] = '100';
-    ENV.web3ProviderUrl = 'http://localhost:7545';
+    ENV.contractMetadata['networkId'] = '42';
```

Comment:
actually I am not sure about this one. by default we run the local ganache network as network ID 100. 
This change requires to always provide that network ID for development with a local network. 
maybe we can add some npm scripts for that? 
`npm run local` `npm run kovan` or something similar?

Reply:
The reason for this is that the default is to not develop contracts, when developing the Web UI, so one shouldn't have to deal with ganache and have to use minimal seed data by default. Scripts would be nice, but I don't think it should be part of the same PR.